File: C/EvenOdd.c
/*
	Write a program to accept a number and find whether 
	it is even or odd.
*/

#include<stdio.h>
#include<stdlib.h>

int main()
{
	int a;
	
	//clrscr();	//Not a part of standard library
	
	system("clear");	//Standard
	
	printf("Enter a number: ");
	scanf("%d", &a);
	
	if(a % 2 == 0)
		printf("This number is even.\n");
	else
		printf("This number is odd.\n");
		
	//getch();
	
	return 0;
}

File: Java/EvenOdd.java
import java.util.Scanner;
class EvenOdd {
	public static void main(String[] args) {
		int n;
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter a number: ");
		n = sc.nextInt();
		if(n%2==0) {
			System.out.println(n + " is an even number.");
		}
		else {
			System.out.println(n + " is an odd number.");
		}
	}
}
